app/controllers/vendor_api/vendor_api_controller.rb

```diff
@@ -3,6 +3,7 @@ class VendorAPIController < ActionController::API
     include ActionController::HttpAuthentication::Token::ControllerMethods
     include LogQueryParams
 
+    rescue_from StandardError, with: :internal_server_error
```

Comment:
I think this will prevent the errors from reaching Sentry, can you check?

Reply:
:+1: I don't think they will reach Sentry from what I can see here https://gitlab.com/gitlab-org/gitlab-foss/-/issues/21043 and here https://github.com/getsentry/raven-ruby/search?q=rescue_from&unscoped_q=rescue_from. I'll ensure this happens

Reply:
@tijmenb https://github.com/DFE-Digital/apply-for-teacher-training/pull/2706/commits/af66c5ae6d0ff0e43dcf3343a01a27b1fecac37d pushes exceptions to Sentry.
It will also raise and return if in development mode as mentioned.